README.md

```diff
@@ -94,6 +98,47 @@ source ./bootstrap.sh -prod --auth my-orgs-creds.base64 --directory /opt/foo --f
 ./download-results.sh
  ```
 
+Until:
+
+If you only want claims data updated or filed before a certain date use the `--until` parameter. The expected format follows the typical
+ISO date time format of `yyyy-MM-dd'T'HH:mm:ss.SSSXXX+/-ZZ:ZZ`
+
+For requests using FHIR R4, a default `_until` value is supplied if one is not provided. The value of the default `_until`
+parameter is set to the current date and time.
+
+Examples:
```

Comment:
There is a space between the other headings and their contents. Do we want to do the same here for consistency? E.g., Files, Limitations, Example